README.md

```diff
@@ -1 +1,70 @@
-# webpack
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+ToDo App
+> A todo webapp configured with webpack
+
+
+## Built With
+
+- Major languages
+  -Java Script 
+HTML 
+- Frameworks
+- Technologies used
```

Comment:
- Please, kindly remove the "Framework" as you've not used one. You can also remove the Technologies or simply add the technologies you've used.